lib/v1/api.js

```diff
@@ -127,12 +127,20 @@
 
             Gratipay.users.getPublic(username, function(data) {
                 element.setAttribute('data-gratipay-readystatus', 'ready');
-                text('receiving', (data.receiving)? '$' + data.receiving : 'anonymously');
+                // Solve lack of two significant digits in some cases (http://stackoverflow.com/a/4937261/2152245)
+                text('receiving', (data.taking)? '$' + parseFloat(data.taking).toFixed(2) : '$' + parseFloat(data.receiving).toFixed(2));
                 text('username', data.username);
                 text('my-tip-button', data.my_tip == 'self' ? 'You!' : (+data.my_tip > 0 ? data.my_tip : 'Gratipay')); // jshint ignore:line
                 text('goal', '$' + data.goal);
                 text('giving', (data.giving)? '$' + data.giving : 'anonymously');
-                link('profile-link', gratipayURI + encodeURIComponent(data.username) + '/');
+
+                // If username exists, it's a ~user, not a team. Build URL accordingly.
+                if (data.username) {
+                    link('profile-link', gratipayURI + encodeURIComponent(data.username) + '/');
```

Comment:
I'm not sure how those rewrites end up working. I know that if I don't put a ~ in, it still works. If it is better to include the ~, then we can do that.